AHEAD, MRIQC, Brainhack Cloud

smoia opened the pull request:
Hello submitters!

@27-apizzuti @sdres @satra @katjaq @r03ert0 @piloubazin
@jacobsanz97 @mtorabi59
@stebo85 @complexbrains @PeerHerholz @Remi-Gau @SamGuay @likeajumprope 

could you please check that your section is as you expect it?
Brainhack Cloud team, would you be able to provide a .bib reference file, please?

You can find the built pdf [here](https://github.com/ohbm/hackathon2022proceedings/suites/7407158625/artifacts/302009896)

complexbrains commented:
I can only see the paragraphs are not spaced between, but I can see it is due to rendering from a doc format, so I guess we will have to accept it. Thank you for the initiative 🚀 .

piloubazin commented:
Our figure numbers are not correct (6 instead of 4), looking at the code it's likely because we and the open cloud folks both kept the \label{fig:test}. Replacing by a different name in \label{} and \ref{} should fix it. Thanks for your work putting this all together!

smoia commented:
@complexbrains and @piloubazin, feel free to make the right changes and create a PR to this branch!

smoia commented on summaries/ahead-project.tex:
```suggestion
	\label{fig:ahead}
```

smoia commented on summaries/brainhack-cloud.tex:
```diff
@@ -0,0 +1,38 @@
+    \includegraphics[width=0.5\textwidth]{brainhack_cloud.png}
+    \caption{A team of brainhack volunteers, applied for Oracle Cloud Credits to support open source projects in and around brainhack with powerful cloud resources on the Oracle Cloud: https://brainhack.org/brainhack\_cloud/
+    }
+    \label{fig:test}
```
```suggestion
    \label{fig:cloud}
```

smoia commented on summaries/ahead-project.tex:
```diff
@@ -0,0 +1,37 @@
+
+\subsection{Results}
+
+Visualization and annotation of large neuroimaging data sets can be challenging, in particular for collaborative data exploration. Here we tested two different infrastructures: BrainBox \url{https://brainbox.pasteur.fr/}, a web-based visualization and annotation tool for collaborative manual delineation of brain MRI data, see e.g. \parencite{heuer_evolution_2019}, and Dandi Archive \url{https://d ...
```
```suggestion
Visualization and annotation of large neuroimaging data sets can be challenging, in particular for collaborative data exploration. Here we tested two different infrastructures: BrainBox \url{https://brainbox.pasteur.fr/}, a web-based visualization and annotation tool for collaborative manual delineation of brain MRI data, see e.g. \parencite{heuer_evolution_2019}, and Dandi Archive \url{https://dandiarchive.org/}, an online repository of microscopy data with links to Neuroglancer \url{https://github.com/google/neuroglancer}. While Brainbox could not handle the high resolution data well, Neuroglancer visualization was successful after conversion to the Zarr microscopy format (fig.~\ref{fig:ahead}A).
```

smoia commented on summaries/ahead-project.tex:
```diff
@@ -0,0 +1,37 @@
+
+Visualization and annotation of large neuroimaging data sets can be challenging, in particular for collaborative data exploration. Here we tested two different infrastructures: BrainBox \url{https://brainbox.pasteur.fr/}, a web-based visualization and annotation tool for collaborative manual delineation of brain MRI data, see e.g. \parencite{heuer_evolution_2019}, and Dandi Archive \url{https://d ...
+
+To help users explore the original high-resolution microscopy sections, we also built a python notebook to automatically query the stains around a given MNI coordinate using the Nighres toolbox~\parencite{huntenburg_nighres_2018} (fig.~\ref{fig:test}B).
```
```suggestion
To help users explore the original high-resolution microscopy sections, we also built a python notebook to automatically query the stains around a given MNI coordinate using the Nighres toolbox~\parencite{huntenburg_nighres_2018} (fig.~\ref{fig:ahead}B).
```

smoia commented on summaries/ahead-project.tex:
```diff
@@ -0,0 +1,37 @@
+
+To help users explore the original high-resolution microscopy sections, we also built a python notebook to automatically query the stains around a given MNI coordinate using the Nighres toolbox~\parencite{huntenburg_nighres_2018} (fig.~\ref{fig:test}B).
+
+For the cortical profile analysis we restricted our analysis on S1 (BA3b) as a part of the somato-motor area from one hemisphere of an individual human brain. S1 is rather thin (\(\sim\)2mm) and it has a highly myelinated layer 4 (see arrow fig.~\ref{fig:test}C). In a future step, we are aiming to characterize differences between S1 (BA3b) and M1 (BA4). For now, we used the MRI-quantitative-R1 co ...
```
```suggestion
For the cortical profile analysis we restricted our analysis on S1 (BA3b) as a part of the somato-motor area from one hemisphere of an individual human brain. S1 is rather thin (\(\sim\)2mm) and it has a highly myelinated layer 4 (see arrow fig.~\ref{fig:test}C). In a future step, we are aiming to characterize differences between S1 (BA3b) and M1 (BA4). For now, we used the MRI-quantitative-R1 contrast to define, segment the region of interest and compute cortical depth measurement. In ITK-SNAP \parencite{Yushkevich2006} we defined the somato-motor area by creating a spherical mask (radius 16.35mm) around the ‘hand knob’ in M1. To improve the intensity homogeneity of the qMRI-R1 images, we ran a bias field correction (N4BiasFieldCorrection, \parencite{Cox1996}). Tissue segmentation was restricted to S1 and was obtained by combining four approaches: (i) fsl-fast \parencite{Smith2004} for initial tissues probability map, (ii) semi-automatic histogram fitting in ITK-SNAP, (iii) Segmentator \parencite{Gulban2018}, and (iv) manual editing. We used the LN2\_LAYERS program from LAYNII open source software \parencite{Huber2021}  to compute the equi-volume cortical depth measurements for the gray matter. Finally, we evaluated cortical depth profiles for three quantitative MRI contrasts (R1, R2, proton density) and three microscopy contrasts (thionin, bieloschowsky, parvalbumin) by computing a voxel-wise 2D histogram of image intensity (fig.~\ref{fig:ahead}C). Some challenges are indicated by arrows 2 and 3 in the lower part of fig.~\ref{fig:ahead}C.
```

stebo85 commented:
Dear @smoia,

The citations in the brainhack cloud section don't look right:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/4021595/179863013-bd153a13-4889-432f-9b71-e41c5fb7ee45.png">

Can you fix this or merge this pull request so we can propose changes?

Also the URLs seem to be too long for the latex template:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/4021595/179863088-95c34bef-7636-49b0-b05e-7ebd5cca6b23.png">

Is there any trick to break a link over a linebreak?

smoia commented:
Hello @stebo85 !
I rebased this PR onto main and fixed the references across the two BHC files. You can find the latest version [here](https://github.com/ohbm/hackathon2022proceedings/suites/7434972620/artifacts/303870599)

Do not worry too much about the hyperlinks breaking pages - this is not necessarily the final template, just a working one.
